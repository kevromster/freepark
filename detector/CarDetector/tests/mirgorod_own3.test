config = camera_nsk_mirgorodskaya5.txt
input = input/mirgorod_own3.png
expected = expected/mirgorod_own3.out
