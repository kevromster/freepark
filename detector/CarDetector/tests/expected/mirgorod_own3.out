[
parking id: 2
spots count: 4

{ lat: 54.85346; lon: 83.040011; complexity: NEWBIE }
{ lat: 54.853453; lon: 83.039974; complexity: NEWBIE }
{ lat: 54.853447; lon: 83.039937; complexity: NEWBIE }
{ lat: 54.85344; lon: 83.0399; complexity: NEWBIE }
]

[
parking id: 1
spots count: 2

{ lat: 54.8535; lon: 83.040389; complexity: NEWBIE }
{ lat: 54.853532; lon: 83.04036; complexity: NEWBIE }
]

[
parking id: 3
spots count: 11

{ lat: 54.85321; lon: 83.040513; complexity: NEWBIE }
{ lat: 54.853224; lon: 83.040498; complexity: NEWBIE }
{ lat: 54.853238; lon: 83.040483; complexity: NEWBIE }
{ lat: 54.853252; lon: 83.040468; complexity: NEWBIE }
{ lat: 54.853267; lon: 83.040453; complexity: NEWBIE }
{ lat: 54.853281; lon: 83.040438; complexity: NEWBIE }
{ lat: 54.853295; lon: 83.040423; complexity: NEWBIE }
{ lat: 54.853309; lon: 83.040408; complexity: NEWBIE }
{ lat: 54.853323; lon: 83.040393; complexity: NEWBIE }
{ lat: 54.853337; lon: 83.040377; complexity: NEWBIE }
{ lat: 54.853351; lon: 83.040362; complexity: NEWBIE }
]

